### ISSUE TITLE
Simplify configv2.json's structure

### ISSUE THREAD

**Issue by amykglen** [RTXteam/RTX#1607]
due to changes to ARAX over the last year-ish, `configv2.json` is feeling very repetitive. the current layout is like this:

* Global
    * mysql_feedback
    * mysql_semmeddb
    * mysql_umls
    * cohd_database
    * graph_database
    * log_model
    * curie_to_pmids
    * node_synonymizer
    * rel_max
    * map_txt
    * dtd_prob
    * kg2c_sqlite
    * kg2c_meta_kg
* Contextual
    * Production
        * neo4j
        * plover
        * cohd_database
        * graph_database
        * log_model
        * curie_to_pmids
        * node_synonymizer
        * rel_max
        * map_txt
        * dtd_prob
        * kg2c_sqlite
        * kg2c_meta_kg
    * KG2
        * neo4j
        * plover
        * cohd_database
        * graph_database
        * log_model
        * curie_to_pmids
        * node_synonymizer
        * rel_max
        * map_txt
        * dtd_prob
        * kg2c_sqlite
        * kg2c_meta_kg
    * KG2c
        * neo4j
        * plover
        * cohd_database
        * graph_database
        * log_model
        * curie_to_pmids
        * node_synonymizer
        * rel_max
        * map_txt
        * dtd_prob
        * kg2c_sqlite
        * kg2c_meta_kg
    * KG2-test
        * neo4j
        * plover
        * cohd_database
        * graph_database
        * log_model
        * curie_to_pmids
        * node_synonymizer
        * rel_max
        * map_txt
        * dtd_prob
        * kg2c_sqlite
        * kg2c_meta_kg
    * local
        * neo4j
        * plover
        * cohd_database
        * graph_database
        * log_model
        * curie_to_pmids
        * node_synonymizer
        * rel_max
        * map_txt
        * dtd_prob
        * kg2c_sqlite
        * kg2c_meta_kg

(each of the innermost slots listed contains a database path and/or username and/or password, which I'm omitting here)

since ARAX now always uses KG2c (via plover), the `Production`, `KG2`, and `KG2c` subsections in the `Contextual` section always contain exactly the same info with the exception of the neo4j info for KG2 vs. KG2c, which is only used by database build processes. in addition, I think the `KG2-test` and `local` sections are unsused (the method of using a `config_local.json` works great for all testing purposes I know of). 

so I think there's not really a point in having a `Contextual` area anymore? all of the slots appearing in each of the Contextual subsections could just appear once under `Global` (or I guess we wouldn't even need to label a `Global` area anymore either, since everything is now "global")

so basically I think the entire layout of `configv2.json` could be something like this:

* mysql_feedback
* mysql_semmeddb
* mysql_umls
* neo4j
    * KG2
    * KG2c
* plover
* cohd_database
* graph_database
* log_model
* curie_to_pmids
* node_synonymizer
* rel_max
* map_txt
* dtd_prob
* kg2c_sqlite
* kg2c_meta_kg

(where each of these slots contains its usual database path and/or password and/or username info)

**Comment by edeutsch** [comment]
agreed

**Comment by amykglen** [comment]
I just caught up on all the traffic in the ITRB channel. @edeutsch - maybe we should just merge `config` into `master` by Tuesday (as Ke suggested). that should avoid the branch-related issue they're running into. and since `config` seems to be working fine on `/test` it seems like it's ok to merge to `master`? (probably after removing your Dockerfile change in the `config` branch)

**Comment by edeutsch** [comment]
agree 100%
I just reversed by Dockerfile commit
Now it's my turn to check out for a while.
Should we plan on this for Monday?

**Comment by amykglen** [comment]
yep, sounds good!

**Comment by amykglen** [comment]
anytime after 2:30pm today works fine for me for merging `config` into `master`, @edeutsch

**Comment by amykglen** [comment]
current status here: we've merged `config` into `master` and rolled the changes out to `/test`, `/beta`, `/NewFmt`, `/kg2beta`, and `/kg2NewFmt`. 

we'll wait for word from ITRB before merging `master` into `itrb-test` and `production`.

`cicd.rtx.ai` isn't _quite_ happy with the changes - believe I have a fix though (will push shortly).

**Comment by amykglen** [comment]
current status: the config overhaul has now been merged into all branches and rolled out to all of our endpoints and all ITRB instances successfully. it's also working on cicd.rtx.ai.

last remaining pieces for me are updating the ARAX wiki and updating the KG2 rollout checklist with the new procedure.

eventually we can delete `configv2.json` from araxconfig.rtx.ai, but probably no rush on doing that (some dev branches are likely still using the old config system).

**Comment by edeutsch** [comment]
Big thanks, @amykglen ! This really tidies the config up substantially!
